.env

```diff
@@ -0,0 +1,9 @@
+DOCKERHUB_NAME=has_many_blogs
```

Comment:
You'll need to use `git rm .env` to remove the file first then commit that change for this file to disappear from this PR. Make sure you save the contents of the file before you run the git rm command since it will delete the file locally on your laptop. So you're basically deleting it through git then adding it again.